android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -235,31 +240,53 @@ public void onMethodCall(MethodCall call, Result result) {
         }
     }
 
-    private void sendEvent(String eventName, @Nullable Map<String, Object> params) {
```

Comment:
this was only used by the updatedPurchaserInfo listener, so I moved it in the new method `setupUpdatedPurchaserInfoListener` and added a null check